.github/workflows/pr_verification.yml

```diff
@@ -395,6 +395,38 @@ jobs:
         uses: gradle/gradle-build-action@v2
         with:
           arguments: spec -Dspec.test.client.host=candlepin
+          
+      - name: Collect docker logs on failure
+        if: failure()
+        uses: jwalton/gh-docker-logs@v2
```

Comment:
@nikosmoum what is the policy for incorporating unverified actions/dependencies like this? From what I am seeing, this looks to be the only action that will pull the logs from docker containers, but I'm not sure what kind of risk we are taking here by using this action.

Reply:
Done, with this [solution](https://github.com/candlepin/candlepin/pull/4197#discussion_r1321662594)